graphbolt/src/isin.cc

```diff
@@ -16,6 +20,12 @@ namespace sampling {
 
 torch::Tensor IsIn(
     const torch::Tensor& elements, const torch::Tensor& test_elements) {
+  if (utils::is_accessible_from_gpu(elements) &&
```

Comment:
I would rename the IsIn to IsInCPU, and create a IsIn to do just dispatch.
IsIn() {
  if (blabla) {
    GRAPHBOLT_DISPATCH_CUDA_ONLY_DEVICE(...)
  } else {
    return IsInCPU(...)
  }
}